dataland-frontend/tests/e2e/specs/high/Prepopulation.ts

```diff
@@ -4,8 +4,8 @@ const chunkSize = 40;
 
 describe("Population Test", { defaultCommandTimeout: Cypress.env("PREPOPULATE_TIMEOUT_S") * 1000 }, () => {
   let companiesWithData: Array<{ companyInformation: CompanyInformation; euTaxonomyData: EuTaxonomyData }>;
-  const teaserCompanies: Array<{ companyIds: string }> = [];
-  let teaserCompaniesPermIds: Array<{ permId: string }> = [];
+  const teaserCompanies: Array<string> = [];
+  let teaserCompaniesPermIds: Array<string> = [];
 
   if (Cypress.env("REALDATA")) {
```

Comment:
Ich würde hier ehr wechseln auf `if (Cypress.env("REALDATA") === true)`. Cypress parsed zwar boolean werte in ENVS von string zum tatsächlichen boolean (damit würde dass dan ja funktionieren). Wenn aber REALDATA kein boolean ist (weil die ENV variable z.B. irgendein anderen String enthält) dann wertet dieses if aufgrund der JS-Logik auch zu true aus und das ist ja eigentlich nicht das erwartete Verhalten.

Reply:
fixed

Reply:
habe aber 
if (Cypress.env("REALDATA") === "true") genommen - dann wird gar nüx geparsed, dann wird der String genommen wie er ist.

Reply:
Eben weil Cypress aber booleans automatisch parsed. Klappt das bei mir mit deiner Lösung leider nicht:
Testszenario: 
![image](https://user-images.githubusercontent.com/108792843/183617867-84dbc090-b312-4afc-a81f-b5012d72fcfa.png)
Command: 
![image](https://user-images.githubusercontent.com/108792843/183617920-565581f6-e3a9-42f8-a186-5f5a603c90b2.png)
In der Konsole wird "DOING STUFF WITHOUT REAL DATA" ausgegeben. Es muss wirklich `===true` sein.

Reply:
(Da klappt dann auch mein Test)

Reply:
gut gesehen. Fixed.

Reply:
Ok